sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/MesosResourcePool.java

```diff
@@ -154,10 +163,14 @@ public Offer getOffer() {
             if (atomicResources == null) {
                 LOGGER.info("Offer lacks any atomic resources named {}", resourceName);
             } else {
+                String desired = spec.getValue().toString();
+                if (spec.getProfiles() != null) {
+                    desired += "(profiles: " + spec.getProfiles() + ")";
```

Comment:
Could do `desired += String.format(" (profiles: %s)", spec.getProfiles())` here (with a leading space to separate from the value)